src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: $duration;
```

Comment:
you need to list all styles for which you apply transition, in your case it's transform and color